posthog/models/filters/filter.py

```diff
@@ -36,7 +37,207 @@
 from posthog.utils import relative_date_parse
 
 
-class Filter(PropertyMixin):
+class DisplayMixin:
```

Comment:
I fiddled with a factory pattern here but resorted to doing a more straightforward pattern for now